### DISCUSSION TITLE
Subtract multiple items

### DISCUSSION THREAD

**Discussion by aestheticaesthetics** [Budibase/budibase#3284] [Help]
Hi I am looking for a way to subtract something like a - b - c - d but subtract currently allows just a - b in tables (one cell minus multiple cells).

Maybe it is just me being new to Budibase and not finding a way, but if there is a way I would be thankful to know it and if there is not, I am glad to submit this as a feature

Thanks and kind regards!

**Comment by mjashanks** [comment]
Just to clarify what you mean...
- You have 4 numbers columns, a,b,c,d
- In a single row, you want to perform the calculation `(a - b -c - d)`
- What do you want to do with the result? Display on screen?

I can use Javascript binding to display the result on screen (see video below). It's possible with the `{{ handlebars }}` syntax as well, but a lot messier!

https://user-images.githubusercontent.com/3524181/140738087-085d581c-fc25-46db-8894-f788ae59326c.mp4

**Reply by aestheticaesthetics** [reply]
Thanks for your answer!

Your clarification is right, but I don't want to display the result on screen but instead have it as a entry in a new column using the "Formula" option in a database (budibase DB). (So Column A - Column B - Column C - Column D = Value of Column E)

**Reply by mjashanks** [reply]
Ah yes - we haven't yet added javascript support for the formula field.

So, until we do... it will look like

```
{{ subtract (subtract (subtract Number A Number B) Number C) Number D }}
```

The order of the number in the above is important!
